client/app/bundles/course/survey/pages/SurveyIndex/__test__/NewSurveyButton.test.jsx

```diff
@@ -38,7 +37,9 @@ afterEach(() => {
 });
 
 describe('<NewSurveyButton />', () => {
-  it('injects handlers that allow surveys to be created', () => {
+  // start_at date field seems to not be updated
```

Comment:
Is this a bug?

Reply:
Not really. I think it's with the new datetime picker component. Need to figure how to change it in test at a later time..